calendar/controllers/auth.controller.js

```diff
@@ -0,0 +1,69 @@
+const jwt = require('jsonwebtoken');
+
+// TODO - use db
+const tokenList = {};
+const revokedAccessTokens = {};
+
+module.exports.revokedAccessTokens = revokedAccessTokens;
+
+module.exports.login = (req, res) => {
+	const body = req.body;
+	const user = {
+		email: body?.email,
+		name: body?.name,
+	};
+	const token = jwt.sign(user, process.env.TOKEN_SECRET, {
```

Comment:
Please add all required env vars to readme file

Reply:
Added README.md file